components/server/src/ome/logic/RenderingSettingsImpl.java

```diff
@@ -93,6 +95,15 @@
     /** Reference to the service used to retrieve the pixels metadata. */
     protected transient IPixels pixelsMetadata;
  
+    /** Reference to the raw pixels store. */
+    private RawPixelsStore rawPixStore;
+    
+    /** Pixel types we can't use the global min/max as channel start/end window **/
```

Comment:
the same will be true for any pixels type if no stats have been calculated so maybe we want to support all pixel types.